docs/risk/risk-faq.md

```diff
@@ -58,15 +58,15 @@ There're clear differences between EigenLayer and rehypothecation.
 
 **If the operator I delegate to is honest, will I be slashed?**
```

Comment:
I'd like to just delete the 3 questions about slashing here

Reply:
got agreement on this offline https://github.com/Layr-Labs/eigenlayer-docs/pull/59/commits/977ccdc511c82b7baa9cec05a2fee8a3fec7bf09